hosting_docs/source/support/1-index.rst

```diff
@@ -0,0 +1,199 @@
+      iii) Tools such as statuspage.io
+
+2. Manage the issue
+
+   a) Response manager or support lead should periodically check in with the developer lead to understand the status and make sure the developer lead has all the support to resolve the issue in a timely manner.
+   b) Post updates on the communication channels regarding the status and ETA.
+
+3. After the issue is resolved
+
+   a) Announce that the issue is resolved on various communication channels
+   b) Take down any banners or update tools such as statuspage.io
+   c) Change the priority of the ticket from P1 to other appropriate priority.
+   d) Update the status of the ticket to ‘Pending Retro’
+
+4. Doing a Retrospective
+
+   a) Ask the developer lead to create a retrospective document that details the root cause of the issue and steps to be taken to prevent such issue from repeating in the future. The developer can use techniques such as `Five Whys <https://en.wikipedia.org/wiki/Five_whys>`_ to do the retrospective.
+   b) Schedule a Retrospective meeting with a wider team to discuss the retrospective and do a postmortem analysis on the ticket to arrive at a comprehensive list of action items to prevent such issues from repeating and make process related improvements to minimize the resolution time.
+
+
+The main difference between a P1 and P2 issue is the urgency with which the issue needs to be resolved. The same process is recommended for P2 issues with relaxations in urgency which means it may not need as frequent and close monitoring as P1.
+
+Advanced Support Process and Tools
+``````````````````````````````````
+Programs that are very large scale could produce a very high volume of support tickets that need to be resolved under SLAs. This requires more advanced support systems to be setup at multiple levels of the program in an escalating manner. This often needs to be planned as a core facet of the program from the ground up. A support system at this level usually consists of
+
+* Issue Tracker Applications to supervisors to support FLWs
+* Helpdesks at District/Block level and escalation process
+* Program level support team at the top
+* View into SLAs
+
+There is no general setup that can be recommended to all the projects as each program has different needs at scale.
+
+..
+    _To add: Dimagi offers <todo; link to support system setup addon package> for this reason. If you require help setting up such system, please contact our delivery team to setup a support system for your project.)
+
+---------------------------------------
+Support System Implementation checklist
+---------------------------------------
+As discussed in :ref:`components-of-support-system`, in order to implement a good support system all of the necessary components need to be in place. You can use the below checklist to make sure you have a robust support system in place.
```

Comment:
This :ref: does not render nicely in GitHub, but `make html` locally renders it correctly.